src/_guides/language/effective-dart/documentation.md

```diff
@@ -209,28 +209,28 @@ makes an API easier to learn.
 If you surround things like variable, method, or type names in square brackets,
 then dartdoc will look up the name and link to its docs.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/docs_good.dart (identifiers)"?>
 {% prettify none %}
-Throws a [StateError] if ...
-
-similar to [anotherMethod], but ...
+/// Throws a [StateError] if ...
+/// similar to [anotherMethod], but ...
 {% endprettify %}
-</div>
 
 You can also link to constructors using `new` followed by the constructor:
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/docs_good.dart (new)"?>
 {% prettify none %}
-To create a point, call [new Point] or use [new Point.polar] to ...
+/// To create a point, call [new Point] or use [new Point.polar] to ...
```

Comment:
I added leading `///`, otherwise the prettifier would colorize the content of the comments (as if it were code).